### ISSUE TITLE
[Unexpected Behavior] The day the music died - Music freezes and stops when a toasty plays sound at the same time that a Windows application plays a sound

### ISSUE THREAD

**Issue by LightningSkunk** [TeamRizu/OutFox#71]
4.6.1a / 4.6.2pre_0815, likely others.

Probably uncommon, and I'm not sure that anything can be done about this until able to switch from DirectSound, but the song music freezes and stops (or disappear-crashes Outfox) when a toasty plays sound at the same time that a background Windows application plays a sound (surprise, it was Discord bablooping). 

First time: Sound freeze, music dies, then Outfox disappear-crashed
Second time (tried to reproduce): sound freeze and the music died but Outfox did not crash out.
Negative control test: Mute Discord, make combo so Dutch Clog Man make silly noise a lot (no issues)

Will try to reproduce a third time and upload logs if needed.

**Comment by Scraticus** [comment]
need logs to this one!

**Comment by LightningSkunk** [comment]
Oh good, it looks like I actually DID grab the logs from the second instance. This one was from 4.6.0 it looks like (thought it was 4.6.1, too many installed versions).

Need logs from a 4.6.2 instance?

Tasty soft freeze:
[info_toasty_softFreeze.txt](https://github.com/TeamRizu/OutFox/files/5081261/info_toasty_softFreeze.txt)
[log_toasty_softFreeze.txt](https://github.com/TeamRizu/OutFox/files/5081262/log_toasty_softFreeze.txt)

**Comment by Scraticus** [comment]
@LightningSkunk These logs are from 22nd of July's build? Can you confirm your Antivirus hasn't done any weirdness, as this is from an older build

**Comment by LightningSkunk** [comment]
Just using Windows Defender, I don't think it's interfering like Norton or McAfee might. 

I might need someone to DM spam me while I try to reproduce on 4.8,2.

**Comment by LightningSkunk** [comment]
Closing this issue.
I dunno what was up with my setup yesterday but I could not reproduce this a third time for the life of me on either version. The best I got was an audio stutter on the toasty audio clip. I even joined several weeb servers and cranked notifications to everything and the cacophony of waifu bots, triggered a disk backup, a virus scan, and even opened discord in the browser as well as the windows app. The full virus scan managed to hang everything but resumed normally.

And no, I didn't download more RAM - Moru and Jose
